### PR TITLE
Speed rgbtoyuvtest up

### PR DESCRIPTION
**Pull request by y-guyon**
Please review only the second commit.

### CODE REVIEW

File: tests/gtest/avifrgbtoyuvthreadingtest.cc [hunk 1]
```diff
@@ -0,0 +1,124 @@
+// Copyright 2022 Google LLC
```
Comment (vrabaud) [inline tests/gtest/avifrgbtoyuvthreadingtest.cc:1]: 2024

Comment (y-guyon) [reply tests/gtest/avifrgbtoyuvthreadingtest.cc:1]: The file was copied in the first commit so we should keep the copyright as is I think.

**Comment by vrabaud** [comment]
Please keep the removed test: maybe DISABLE them.

**Comment by y-guyon** [comment]
> Please keep the removed tests: maybe DISABLE them.

They are already tested with min/max rgb values:

https://github.com/AOMediaCodec/libavif/blob/b10d2697e9ed2fb09cb722335ff4342c353612b8/tests/gtest/avifrgbtoyuvtest.cc#L296-L349

I do not think it is necessary to add a DISABLED test suite instantiation.

**Review by vrabaud — APPROVED** [review]

**Pull request merged**